docs/conf.py

```diff
@@ -55,3 +55,9 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
```

Comment:
This is currently failing on the CI coz I think we don't have anything on that path:

```
html_static_path entry '_static' does not exist
```

Reply:
I'm gonna add a .gitignore to make sure that file is included.